app/celery/tasks.py

```diff
@@ -291,7 +297,9 @@ def send_email(service_id, notification_id, subject, from_address, encrypted_not
 def send_sms_code(encrypted_verification):
     verification_message = encryption.decrypt(encrypted_verification)
     try:
-        firetext_client.send_sms(verification_message['to'], verification_message['secret_code'])
+        firetext_client.send_sms(
+            verification_message['to'], verification_message['secret_code'], 'send-sms-code'
```

Comment:
String "send-sms-code" to identify the firetext callback as different to a notification one.